install.sh

```diff
@@ -224,10 +224,7 @@ if [ "$1" = "install" ]; then
       mkdir -p RPMS
       
       # Move rpm files to RPMS folder inside the linux-tkg folder
-      mv ~/rpmbuild/RPMS/x86_64/* "$_where"/RPMS/
-
-      #Clean up the original folder, unneeded and takes a lot of space
-      rm -rf ~/rpmbuild/
+      mv ~/rpmbuild/RPMS/x86_64/*tkg* "$_where"/RPMS/
```

Comment:
That works, albeit removing the tkg build files was a good idea 😝

OK I found that it's possible to use `RPMOPTS='--define "_topdir /tmp/test"' make rpm-pkg` to build in a directory that isn't ~/rpmbuild, so I think it would be better to use the previous mv and rm -rf lines, but in a tkg temporary directory inside ~/.cache

Reply:
Okay thanks for the tip! I will implement that and get back to you :D

Reply:
Is there a problem with using `/tmp` ? Since it's in RAM, I guess it's better ?

Reply:
`/tmp` might run out of space (on my computer it has 7.8G and it doesn't seem to be enough, although I can increase it because I have 16G of RAM). Some would prefer it (in Arch they would do it via /etc/makepkg.conf), perhaps we can add a config option if there isn't something more standardized

Reply:
The config option makes sense actually, I will do that, I just had an issue too on my VM of 5GB RAM, I though the `rpmbuild` folder takes less than 2GB but apparently not.

Reply:
Do you have any idea how to change the working folder with `deb-pkg` ?

Reply:
Let's see... is it currently building at the kernel directory in `/debian`? It looks hardcoded, starting from `/scripts/package/mkdebian` (`mkdir -p debian/source/`)

Reply:
I guess symlinking the debian directory before `make` would do the trick?

Reply:
The RPMs find themselves in the same folder as the linux-tkg git folder, I don't know exactly where the temporary files are put.

Reply:
You are saying that there's a temporary `debian` directory created in the linux-tkg git folder, and if we create first `make` will write inside it ?

Reply:
Sorry for the late reply. You mean the RPMs don't stay at `?/rpmbuild/RPMS/x86_64/` after `make rpm-pkg`?
Yeah I saw it created a `debian` directory, I believe `make` would write into it without error if we create/symlink it ourselves before it does. I'm going to need a very minimalistic kernel config in order to fully test it myself heh

Reply:
Oh no the RPMs do stay, I was only talking about debs. Don't worry I have everything setup to test it, I will try your hack and see :D

Reply:
Haha yeah. Ok cool, give it a try, check if the `debian` directory isn't deleted automatically after `make`

Reply:
Nah it doesn't work, I don't see any `debian` directory being created, I will just do your first suggestion in `~/.cache` for now.

Reply:
That's odd. I can download the kernel, run `make deb-pkg` after `make menuconfig` and see the `debian` directory shortly after. Maybe it's deleted relatively quick on a deb distro? It contains files such as `control` and `rules`

Reply:
Oh maybe, but while things are compiling, when compared to RPM compilation, the folder isn't there any longer. So it is not some kind of working directory where compilation temp files are kept. But maybe nothing really is kept. The compiler itself does use `/tmp` though.

Reply:
Oh I see. /tmp doesn't run out of space, so might have to resort to using `pyinotify` or something 😅

Reply:
I am not sure I understood you, as far as I understand `/tmp` is on the RAM for systemd distros and can by default fill up to half the amount of the total RAM. So it can run out of space and I already got that error. The thing is, when I said "`/tmp` is used by the compiler" I meant for a small amount of temporary intermediary compilation files. So nothing much. Why would I need to resort to `pyinotify` ?

Reply:
I just find it weird that building for rpm needs gigabytes of storage to build and deb not

Reply:
Yea I have noticed that  when I was testing, and honestly I don't know why it produces so many side products, I could do `bin-rpmpkg` which doesn't produces all these files but the kernel-devel package isn't made anymore, needed to compile nvidia modules for example :/